### PR TITLE
feat: Updates for deploy previews

### PR DESCRIPTION
**Pull request by guy-lawrance-manypets**
- Removed some debug logging from last iteration
- Increased wait for service stability 10min -> 15 mins
- Added support for env vars for ECR image services

### CODE REVIEW

**Review by a3m33 — APPROVED** [review]
LGTM

**Review by a3m33 — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -112,6 +115,14 @@ export async function run(): Promise<void> {
         // Memory - 2
         const memory = getInputInt('memory', 2);
 
+        // ENV VARs
+        let config = undefined;
+        if (envText) {
+            const buf = Buffer.from(envText);
+            config = parse(buf);
+            console.log(typeof config, config)
```
Comment (a3m33) [inline src/index.ts]: Can this be removed?

File: src/index.ts [hunk 2]
```diff
@@ -112,6 +115,14 @@ export async function run(): Promise<void> {
         // Memory - 2
         const memory = getInputInt('memory', 2);
 
+        // ENV VARs
+        let config = undefined;
```
Comment (a3m33) [inline src/index.ts]: Should this be a type annotation and then assignment?

File: src/index.ts (hunk 1 above)
Comment (guy-lawrance-manypets) [reply src/index.ts]: Good idea - I don't think we should be logging the env config, unless for debug

**Pull request merged**